src/hooks/filter/useFilterSearchAndSort.js

```diff
@@ -0,0 +1,12 @@
+import { useCallback } from 'react';
+
+export const useSort = (options, handler) => {
+  const setOptionHandler = useCallback((e) => {
+    const result = options.find((item) => item.value === e.target.value);
+
+    if (result) {
+      handler(result);
+    }
+  });
+  return setOptionHandler;
+};
```

Comment:
add array of dependecies to useCallback hook

Reply:
oh, thank you i missed it